djautotask/sync_rest.py

```diff
@@ -610,7 +618,7 @@ def _assign_field_data(self, instance, json_data):
 
     def fetch_sync_by_id(self, instance_id):
         instance = super().fetch_sync_by_id(instance_id)
-        if not instance.status or \
+        if instance and not instance.status or \
```

Comment:
I guess just revert this after you make the changes for the other comment about 404's